aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -32,6 +34,15 @@ module aptos_framework::account {
         new_authentication_key: vector<u8>,
     }
 
+    #[event]
+    struct KeyRotationToMultiPublicKey has drop, store {
+        account: address,
+        verified_public_key_bit_map: vector<u8>,
+        multi_public_key_scheme: u8,
```

Comment:
Is this the threshold? I thought there's only one scheme for multikey

Reply:
No. there is multikey and multiEd.  multiEd is legacy.  

Schemes are as follows
ed25519 - 0
multiEd - 1 
singleKey - 2
multiKey - 3

Reply:
I'm confused - if it's always 3, why does this need to be in the event?

Reply:
it could be 1